plugins/modules/vmware_vc_infraprofile_info.py

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019-2020, Ansible Project
+# Copyright: (c) 2019-2020, Naveenkumar G P <ngp@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: vmware_vc_infraprofile_info
+short_description: List, Export, Validate and import VMware Vcenter infra profile configs.
```

Comment:
I think we need to refresh the `short_description` since this purpose of this module is to expose the information (no import and validation).